common/src/main/java/com/google/udmi/util/CommandLineProcessor.java

```diff
@@ -91,6 +99,10 @@ private void showUsage() {
    */
   public void showUsage(String message) {
     ifNotNullThen(message, m -> System.err.println(m));
+    ifNotNullThen(this.usageForms, () -> {
+      System.err.println("Usage forms:");
+      this.usageForms.forEach(form -> System.err.printf("\t%s%n", form));
```

Comment:
nit: I generally avoid tabs since they just end up being more confusing (sometimes preserved by cut-and-past, sometimes not, can confuse diff output, etc... ) -- so I'd just expand with spaces...